state/secrets.go

```diff
@@ -1110,11 +1121,16 @@ func (w *consumedSecretsWatcher) loop() (err error) {
 //
 // Obsolete revisions results are "uri/revno" and deleted
 // secret results are "uri".
-func (st *State) WatchObsolete(owner string) (StringsWatcher, error) {
-	if _, err := names.ParseTag(owner); err != nil {
-		return nil, errors.NewNotValid(err, "invalid owner")
+func (s *secretsStore) WatchObsolete(owners []string) (StringsWatcher, error) {
```

Comment:
Could we pass []names.Tag here？ Then no need to validate in both facade and state again.

Reply:
ok